braindecode/augmentation/transforms.py

```diff
@@ -0,0 +1,1000 @@
+            "spherical_splines should be a boolean"
+        self.sensors_positions_matrix = sensors_positions_matrix
+        self.axis = axis
+        self.spherical_splines = spherical_splines
+
+        # override max_degrees value when a magnitude is passed
+        if magnitude is not None:
+            min_val, max_val = mag_range
+            self.max_degrees = magnitude * max_val + (1 - magnitude) * min_val
+        else:
+            self.max_degrees = max_degrees
+
+        # Handle case of 0 degrees
+        if self.max_degrees == 0:
+            operation = identity
+        else:
+            operation = random_rotation
+
+        super().__init__(
+            probability=probability,
+            operation=operation,
+            magnitude=magnitude,
+            mag_range=mag_range,
+            axis=self.axis,
+            max_degrees=self.max_degrees,
+            sensors_positions_matrix=self.sensors_positions_matrix,
+            spherical_splines=self.spherical_splines,
+            random_state=random_state
+        )
+
+
+class RandomZRotation(RandomSensorsRotation):
+    """Interpolates EEG signals over sensors rotated around the Z axis
+    with an angle sampled uniformly between 0 and `max_degree`.
+
+    Suggested in [1]_
+
+    Parameters
+    ----------
+    probability : float | callable
+        Either a float between 0 and 1 defining the
+        uniform probability of applying the operation, or a `callable`
+        (modelling a non-uniform probability law for example) taking no input
+        and returning a boolean.
+    ordered_ch_names : list
+        List of strings representing the channels of the montage considered.
+        Has to be in standard 10-20 style. The order has to be consistent with
+        the order of channels in the input matrices that will be fed to the
+        transform. This channel will be used to compute approximate sensors
+        positions from a standard 10-20 montage.
+    magnitude : float | None, optional
+        Float between 0 and 1 encoding the `max_degree` parameter:
+        ```
+        max_degree = magnitude * mag_range[1] + (1 - magnitude) * mag_range[0]
+        ```
+        Defaults to None (ignored).
+    mag_range : tuple of two floats | None, optional
+        Range of possible values for `max_degree` settable using the magnitude
+        (see `magnitude`). If omitted, the range (0, 30 degrees) will be used.
+    axis : 'x' | 'y' | 'z', optional
+        Axis around which to rotate. Defaults to 'z'.
+    max_degree : float, optional
+        Maximum rotation. Rotation angles will be sampled between `-max_degree`
+        and `max_degree`. Defaults to 15 degrees.
+    spherical_splines : bool, optional
+        Whether to use spherical splines for the interpolation or not. When
+        `False`, standard scipy.interpolate.Rbf (with quadratic kernel) will be
+        used (as in the original paper). Defaults to True.
```

Comment:
This is not exposed I guess?
```suggestion
```

Reply:
You're right, but it's a mistake :) I will fix it